tensorboard/plugins/graph/tf_graph_op_compat_card/tf-graph-op-compat-card.html

```diff
@@ -227,11 +227,6 @@
         _getNode: function(nodeName, graphHierarchy) {
           return graphHierarchy.node(nodeName);
         },
-        _getPrintableHTMLNodeName: function(nodeName) {
```

Comment:
I found this copy of _getPrintableHTMLNodeName() that appears to be unused. I don't think the wbr logic is necessary for the compatibility info section.